askama_shared/src/parser.rs

```diff
@@ -1011,15 +1011,33 @@ fn block_comment<'a>(i: &'a [u8], s: &'a Syntax<'a>) -> IResult<&'a [u8], Node<'
     let mut p = tuple((
         |i| tag_comment_start(i, s),
         opt(tag("-")),
-        take_until(s.comment_end),
+        |mut i| {
```

Comment:
Let's pull this out into a named function, please?

Reply:
Actually had that originally, and removed it 😅

Reply:
Why? :)

Reply:
Personally, I prefer a function because readability. But I removed it, because I was in the mindset of minimizing the change. However, I'm adding it again 😅